### PR TITLE
Fixing correct default values for AntreaConfig template

### PR DESCRIPTION
**Pull request by knabben**
### What this PR does / why we need it
The AntreaConfig generated on Cluster do not have the correct default values, this PR fix the template used to generate the default CR for the CBT

### Which issue(s) this PR fixes

Fixes #

### Describe testing done for PR

A testbed with the new template is created and the default AntreaConfig had the expected flags enabled.

### Release note

```release-note

```

### CODE REVIEW

**Comment by alfredthenarwhal** [comment]
> @knabben:  /test install-vc7
Commit: 67e9d29

Tests failed! Build no: 3583

**Comment by alfredthenarwhal** [comment]
> @knabben:  /test install-vc7
Commit: 7895042

Build failed! Build no: 3585

File: addons/config/expected/cni.tanzu.vmware.com/v1alpha1/antreaconfig/default.yaml [hunk 1]
```diff
@@ -8,18 +8,57 @@ metadata:
 spec:
   antrea:
     config:
-      trafficEncapMode: encap
-      noSNAT: false
-      tlsCipherSuites: TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_256_GCM_SHA384
```
Comment (adduarte) [inline addons/config/expected/cni.tanzu.vmware.com/v1alpha1/antreaconfig/default.yaml:13]: Why is this being removed?

**Comment by adduarte** [comment]
seems there is a bit of changes in this pr to just fix the "NodePortLocal: true", 
perhaps we can narrow this.

File: addons/config/expected/cni.tanzu.vmware.com/v1alpha1/antreaconfig/default.yaml (hunk 1 above)
Comment (knabben) [reply addons/config/expected/cni.tanzu.vmware.com/v1alpha1/antreaconfig/default.yaml:13]: the template here is an old version - this is the default value of the field set by the antrea controller.

**Comment by knabben** [comment]
> seems there is a bit of changes in this pr to just fix the "NodePortLocal: true", perhaps we can narrow this.

We need to enable these feature flags by default on 1.7.1, the other fields are supposed to be filled by the controller.

**Comment by adduarte** [comment]
LGTM. This looks good from teplating point of view. 
please add details about what testing has been done before we can approve. thanks.

**Review by adduarte — CHANGES_REQUESTED** [review]
This looks good but needs testing of antrea to verify how it affectst it.

**Comment by alfredthenarwhal** [comment]
> @knabben:  /test install-vc7
Commit: 6323e16

Tests failed! Build no: 3588

**Review by edwardbadboy — APPROVED** [review]
Thanks for fixing this.

**Comment by alfredthenarwhal** [comment]
> @knabben:  /test install-vc7
Commit: 6323e16

Build failed! Build no: 3596

**Comment by alfredthenarwhal** [comment]
> @knabben:  /test install-vc7
Commit: d0b04d4

Tests failed! Build no: 3607

**Comment by knabben** [comment]
> This looks good but needs testing of antrea to verify how it affectst it.

All tests were conducted internally and confirm the correct flags are enabled.

**Review by imikushin — APPROVED** [review]
LGTM

**Review by adduarte — APPROVED** [review]

**Pull request merged**